manifests/charts/gateway/files/profile-compatibility-version-1.20.yaml

```diff
@@ -0,0 +1,6 @@
+pilot:
```

Comment:
Yeah this is the kind of YAML file I was thinking about earlier. Why not make our code depend on this?